include/wlr/render/pixman.h

```diff
@@ -9,9 +9,30 @@
 #ifndef WLR_RENDER_PIXMAN_H
 #define WLR_RENDER_PIXMAN_H
 
+#include <pixman.h>
 #include <wlr/backend.h>
 #include <wlr/render/wlr_renderer.h>
 
 struct wlr_renderer *wlr_pixman_renderer_create(void);
 
+struct wlr_pixman_texture_attribs {
+	void *data;
+	pixman_image_t *image;
+	const struct wlr_pixel_format_info *format;
```

Comment:
Do callers really need the `struct wlr_pixel_format_info`?

Reply:
No